external/storm-kafka/src/jvm/org/apache/storm/kafka/PartitionManager.java

```diff
@@ -46,6 +46,9 @@
     private final CountMetric _fetchAPIMessageCount;
```

Comment:
since we are refactoring this code. Can we simplify partitionManager constructor instead of the current state we can have few private methods for all the code in the constructor.